Misc/votvlibrary.lua

```diff
@@ -20,13 +20,14 @@ function move_point_contact_solid(x, y, angle, amount)
 	local totalMoved = 0
 	local xx = math.cos(math.rad(angle))
 	local yy = math.sin(math.rad(angle))
-	while totalMoved <= amount do
+	while totalMoved < amount do
 		x = x + xx * 32
 		y = y + yy * 32
-		totalMoved = totalMoved + xx * 32 + yy * 32
-		if totalMoved > amount then
+		totalMoved = totalMoved + 32
```

Comment:
Im guessing 32 here is just 32x32 sprites but still a comment explaining the number